gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-bin.zip
```

Comment:
Updating Gradle to the same version used by Clouddriver https://github.com/spinnaker/clouddriver/blob/master/gradle/wrapper/gradle-wrapper.properties#L3